torch/testing/_internal/common_methods_invocations.py

```diff
@@ -5271,6 +5271,7 @@ def sample_inputs_nll_loss(op_info, device, dtype, requires_grad, **kwargs):
         ((*shape, 1, 2), dict()),
         ((*shape, 1, 2, 3), dict()),
         (shape, dict(weight=make_tensor((num_classes,), device=device, dtype=dtype).abs())),
+        (shape, dict(weight=make_tensor((num_classes,), device=device, dtype=dtype))),
```

Comment:
If we explicitly want to test this for positive and negative weights, maybe we should simply set `low=0` (and remove the `.abs()`) in the line above and here set 

```suggestion
        (shape, dict(weight=make_tensor((num_classes,), high=0, device=device, dtype=dtype))),
```

Reply:
Good point. Corrected!

Reply:
Instead of leaving the `.abs()` call in the line above, could you just set `low=0` in the `make_tensor()` call? Or remove it completely to indicate "any kind of weight".

Reply:
Left one with positive, one with negative and one with mixed.